### ISSUE TITLE
[BUG] Python 2.7 + Windows = password prompting fails

### ISSUE THREAD

**Issue by ronlut** [nishad/udemy-dl#92]
Please prefix your issue with one of the following: **[BUG]** **[PROPOSAL]** **[QUESTION]**.

**In raising this issue, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/nishad/udemy-dl/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed issues](https://github.com/nishad/udemy-dl/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/nishad/udemy-dl/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

**If the project maintainer has any additional requirements, they will be listed here.**

- No additional requirements.

---

So, when using the library on windows with python 2.7 (2.7.10 in my case), password prompt fails with the following message:

```
Traceback (most recent call last):
  File "C:\Python27\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\Rony\Desktop\udemy-dl-master\src\udemy_dl\__main__.py", line 8, in <module>
    main()
  File "udemy_dl\udemy_dl.py", line 680, in main
    password = getpass.getpass(prompt='Password : ')
  File "C:\Python27\lib\getpass.py", line 95, in win_getpass
    msvcrt.putch(c)
TypeError: must be char, not unicode
```

Happens not only here: [example#1](https://reviews.reviewboard.org/r/6983/diff/4#index_header), [example#2](https://bitbucket.org/ZyX_I/gibiexport/issues/2/error-getting-password-on-windows)

I will fix it and create a pull request soon hopefully.

**Comment by nishad** [comment]
Try the latest development version and let me know if it works.

**Comment by nishad** [comment]
Fixed with v0.2.1. Follow steps in https://github.com/nishad/udemy-dl/blob/master/README.md#update to update.

**Comment by anschauhanex** [comment]
still facing the same issue with v0.2.1
